docs/DeveloperGuide.md

```diff
@@ -965,3 +965,22 @@ testers are expected to do more *exploratory* testing.
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Finding sales of drinks
```

Comment:
This section should be above "Saving data"

Reply:
added now